src/ir/item.rs

```diff
@@ -444,7 +444,7 @@ impl ClangItemParser for Item {
         if cursor.kind() == clangll::CXCursor_UnexposedDecl {
             Err(ParseError::Recurse)
         } else {
-            error!("Unhandled cursor kind: {}", ::clang::kind_to_str(cursor.kind()));
+            warn!("Unhandled cursor kind: {}", ::clang::kind_to_str(cursor.kind()));
```

Comment:
We should keep this, and if we don't care about any specific cursor, like `MacroExpansion`, we should do something like:

```rust
match cursor.kind() {
    kind @ CXCursor_MacroExpansion | /* others? */ => debug!("Ignoring cursor kind {}", ::clang::kind_to_str(kind)),
    other => error!("Unhandled cursor kind: {}", ::clang::kind_to_str(kind)),
}
```